pkg/volume/azure_dd/azure_common_linux.go

```diff
@@ -70,17 +70,17 @@ func getDiskLinkByDevName(io ioHandler, devLinkPath, devName string) (string, er
 }
 
 func scsiHostRescan(io ioHandler, exec mount.Exec) {
-	scsi_path := "/sys/class/scsi_host/"
-	if dirs, err := io.ReadDir(scsi_path); err == nil {
+	scsiPath := "/sys/class/scsi_host/"
```

Comment:
Can you also remove this file from https://github.com/kubernetes/kubernetes/blob/master/hack/.golint_failures so that we don't regress in the future?

Reply:
I got it.

Reply:
Hello msau42,
I use golint to test again,
But there are still some errors (don't use an underscore in package name)
Can be remove from the list?

Reply:
yes I think it would be good to resolve all golint errors